src/helpers.rs

```diff
@@ -311,7 +311,7 @@ pub fn process_chunks<T>(receiver: Receiver<(Vec<u8>, usize, bool)>) -> Vec<(T,
 where
     T: ReadPointFromBytes,
 {
-    #[allow(clippy::unnecessary_filter_map)]
+    // TODO: should we use rayon to process this in parallel?
```

Comment:
The process chunks is called from a parallel processor which splits up the entire file to chunks and calls the function and later arranges the results in order, if thats what you were thinking.

Reply:
no I meant to use par_iter here instead of iter. If you only have a few cpus lets say, then you'll only have a few workers, and each will be processing a lot of points, sequentially. Not sure if all the cpus are saturated already... maybe.